lib/migrator/methods/configure.js

```diff
@@ -10,7 +10,13 @@ module.exports = function configure(params) {
 		.then(() => {
 			this.params = {
 				// migrations dir
-				dir: pathUtils.resolve('migrations'),
+				dir: null,
```

Comment:
This will mean having an odd setup like:
When the user specifies only `--migrationExtension ts`
then `--sourceMigrationExtension` is left to be `js` by default, and this is very improbable that the user compiles `js` to `ts`.

Maybe we should not only set `--sourceDir` equal to `dir` by default, but also `--sourceMigrationExtension`.

As for passing only `--sourceMigrationExtension ts` it does seem resonable that `--migrationExtension` is left to be `js` by default.

And one more setup when the user passes only `--sourceDir tsFilesDir` it is not expected that `--dir` will be equal to `migrations`, because you'd rather expect it to be in `build` or `out` directories where your transpiled javascript files reside. Though maybe just documenting this more thoroughly will be enough.

Reply:
Done in https://github.com/okv/east/pull/54/commits/bc6b8ad847d5b3bd97ff2234dc259d6669b9a956

Reply:
Your current implementation (when `dir`, `migrationExtension` are static and `sourceDir`, `sourceMigrationExtension` are equal to `dir` and `migrationExtension` by default) looks good to me. I think it's ok to set `dir` manually (if you don't like default "migrations" dir) when you set `sourceDir`. I suggest to document options default values at east binary option description.

Reply:
You already documented it, nice)